### ISSUE TITLE
wrong relation is specific for integrator to k8s-master

### ISSUE THREAD

**Issue by jeff-hillman** [charmed-kubernetes/kubernetes-docs#540]
The relation shown to get keystone credentials from openstack-integrator to kubernetes-master provided is incorrect.  

it currently says 

```
juju add-relation kubernetes-master:keystone-credentials openstack-integrator:keystone-credentials
```

When in fact it should be 

```
juju add-relation kubernetes-master:keystone-credentials openstack-integrator:credentials
```

**Comment by jeff-hillman** [comment]
sorry, in reference to this page https://ubuntu.com/kubernetes/docs/ldap

**Comment by evilnick** [comment]
This was fixed by #550, should be live in a few hours. Thanks for pointing it out!
